rasterio/plot.py

```diff
@@ -33,38 +33,69 @@
 logger = logging.getLogger(__name__)
 
 
-def show(source, cmap='gray', with_bounds=True):
+def show(source, cmap='gray', with_bounds=True, ax=None, title=None, **kwargs):
```

Comment:
I had to lookup bike-shedding...  but that seems like a reasonable observation.  How about setting cmap=None and using the default colormap, in case someone has set it in their rc params?  Better yet drop it from the explicit parameters altogether.  A user could still pass it if they wanted because it would get passed through with the kwargs.